### DISCUSSION TITLE
How can I find the packages that is the reason that poetry downgrades dependencies?

### DISCUSSION THREAD

**Discussion by Vuizur** [python-poetry/poetry#8302] [Q&A] [answered]
I want to install the package `librosa` on Windows 11 and Python 3.11. When I install it with `pip install`, it works well. When I install it with `poetry add librosa`, it produces an error, because it tries to install an outdated version of llmvlite, where due to a bug no wheels can be built. 

I assume that there is some package in the whole dependency tree that causes this, does anyone know the easiest way to identify it?

Thanks for the help!

**Comment by dimbleby** [comment, marked as answer]
try requiring a higher version of llvmlite, and let poetry tell you what the conflict is

**Reply by Vuizur** [reply]
Thanks!
